### PR TITLE
Fix #1707 x86 build 

### LINKED ISSUE #1707: [bug] build failed on x86-64
uftrace/arch/x86_64/mcount-insn.c:479:3: error: label at end of compound statement
   default:
   ^~~~~~~

### PR DESCRIPTION
**Pull request by yihong0618**
This PR fix build on x86_64 fix #1707

### CODE REVIEW

File: arch/x86_64/mcount-insn.c [hunk 1]
```diff
@@ -476,7 +476,7 @@ static int check_instrumentable(struct mcount_disasm_engine *disasm, cs_insn *in
 		case CS_GRP_PRIVILEGE:
 		case CS_GRP_BRANCH_RELATIVE:
 			return INSTRUMENT_FAIL_NO_DETAIL;
-		default:
+		default:;
```
Comment (namhyung) [inline arch/x86_64/mcount-insn.c:479]: Please don't mix two changes in a commit.  This change should be separate from the Github workflow.  Also you'd better have `break` statement in the following line.

File: .github/workflows/pr-test.yml [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
 jobs:
   unit-test-with-asan:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch:
+          - x86_64
+          - aarch64
+          - arm64
```
Comment (namhyung) [inline .github/workflows/pr-test.yml:17]: Aren't aarch64 and arm64 the same?

**Review by namhyung — COMMENTED** [review]
Please rebase your changes on to the current master (after running `git pull`).  And the commit message should have a proper prefix.  'fix' is not a proper one.  It can be 'build' or 'arch/x86' or 'dynamic'.  In addition, please consider adding the actual compiler error message to see what's going on here since it might depend on the compiler and its version.  It's a convention to indent by two spaces those messages.

**Comment by yihong0618** [comment]
copy that will fix it tonight thanks

File: .github/workflows/pr-test.yml [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+
 jobs:
   unit-test-with-asan:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        arch:
+          - x86_64
+          - aarch64
```
Comment (namhyung) [inline .github/workflows/pr-test.yml:16]: Actually I'm not sure if it's really needed to run the unittest on multiple archs.

File: arch/x86_64/mcount-insn.c [hunk 4]
```diff
@@ -477,6 +477,7 @@ static int check_instrumentable(struct mcount_disasm_engine *disasm, cs_insn *in
 		case CS_GRP_BRANCH_RELATIVE:
 			return INSTRUMENT_FAIL_NO_DETAIL;
 		default:
+			break;
```
Comment (namhyung) [inline arch/x86_64/mcount-insn.c:480]: Please move it after the comment.

File: .github/workflows/pr-test.yml (hunk 3 above)
Comment (namhyung) [reply .github/workflows/pr-test.yml:16]: Could please check the actual result?  It seems it didn't run the test on an ARM (64-bit) machine.

Comment (yihong0618) [reply .github/workflows/pr-test.yml:16]: will check it today,  sorry for the inconvenience

**Comment by yihong0618** [comment]
@namhyung you are right,  the `arch` did not work...
I think it needs a lot of work to test on different arch, so I want to change back and do not change the actions things on different arch, do you agree?


maybe need this:
https://github.com/marketplace/actions/run-on-architecture

**Comment by namhyung** [comment]
I'm ok to drop the github actions and just to fix the build problem.

**Comment by yihong0618** [comment]
Done

**Comment by namhyung** [comment]
For this kind of change, it'd be nice if you could add the actual compiler warning message to the commit like in dae3c721d2d.   Note that you may want to indent them by 2 spaces to prevent accidental commenting out if it starts with the hash (#).   Anyway I prefer having them indented (even without the hash) to indicate they are copied from elsewhere.

**Comment by yihong0618** [comment]
of course, I will learn your commit and revise mine, thanks for the advice again~

**Review by namhyung — APPROVED** [review]
LGTM, thanks for your endurance.

**Comment by honggyukim** [comment]
Hi @yihong0618, it was my mistake. Thanks for the fix!

**Pull request merged**